tests/command/toggleattributecommand.js

```diff
@@ -5,7 +5,7 @@
 
 import Editor from '/ckeditor5/core/editor/editor.js';
 import Document from '/ckeditor5/engine/model/document.js';
-import AttributeCommand from '/ckeditor5/core/command/attributecommand.js';
+import ToggleAttributeCommand from '/ckeditor5/core/command/toggleattributecommand.js';
```

Comment:
Yeah, @pjasiun is right. It should be a verb. Though, we used nouns sometimes for features. But perhaps that was wrong – command is an action + state, so its name should indeed represent that and, also, there should be no command like `image` which insert and edit image at the same time. OTOH, such commands are useful, because they you can use one in your button. Tricky :).

Anyway, `ToggleAttributeCommand` is fine for me. `ToggleCommand` isn't, because it doesn't says what it toggles.